src/network.zig

```diff
@@ -1450,19 +1450,21 @@ pub const Connection = struct { // MARK: Connection
 		self.manager.send(fullData, self.remoteAddress, null);
 	}
 
-	fn receiveKeepAlive(self: *Connection, data: []const u8) void {
+	fn receiveKeepAlive(self: *Connection, data: []const u8) !void {
```

Comment:
As discussed in #1156 I left interface unchanged. Maybe after #1259 we could change it to accept reader instead?

Reply:
Yeah, that's fine for now. #1259 is likely going to do major changes here anyways.